theme/static/js/collaborate.js

```diff
@@ -85,20 +84,30 @@ $(document).on('change', '.btn-file :file', function () {
 });
 
 $(document).ready(function () {
+    set_feedback_visible(false);
     $(".btn-ask-to-join").click(request_join_group_ajax_submit);
     $(".btn-act-on-request").click(act_on_request_ajax_submit);
     $("#txt-search-groups").keyup(function () {
+        set_feedback_visible(false);
+        let num_match_found = 0;
         var searchString = $("#txt-search-groups").val().toLowerCase();
         $(".group-container").show();
         var groups = $(".group-container");
         for (var i = 0; i < groups.length; i++) {
             var groupName = $(groups[i]).find(".group-name").text().toLowerCase();
             if (groupName.indexOf(searchString) < 0) {
                 $(groups[i]).hide();
+            } else {
+                num_match_found++;
             }
         }
+        if (num_match_found == 0 && searchString.trim().length > 0) {
+            set_feedback_visible(true);
```

Comment:
This could be replaced with `$("#id-Group-Search-Result-Msg").show();`

Reply:
precisely.